specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -803,12 +803,12 @@
           "description": "Version of the guest configuration."
         },
         "contentUri": {
-          "readOnly": true,
+          "readOnly": false,
```

Comment:
instead of changing this to `false`, could we just remove this line? The default value of `readOnly` is `false`. We actually do not need to emphasize this here.